CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - When expanding a class with a strategy, you can now create rules where the
   parent is not the class passed to the strategy.
 - The 'get_equations' method now handles multiple parameters
+- an option on `auto_search` to not expand verified classes
```

Comment:
Is this meant to be here, or up in [Unreleased]?

Reply:
fixed